src/cli/commands/start.ts

```diff
@@ -0,0 +1,117 @@
+  rpc: string;
+  configFile: string;
+  key: string;
+  dbValidator: string;
+  loggingLevel: string;
+}
+
+export class StartCommand implements CliCommand {
+
+  public register(commander: CommanderStatic): void {
+    const logger: ILogger = new WinstonLogger();
+
+    commander
+      .command("start")
+      .description("Start lodestar node and bundled validator")
+      .option("--db [db_path]", "Path to Beacon node database")
+      .option("-dc, --depositContract [address]", "Address of deposit contract")
+      .option("-eth1, --eth1RpcUrl [url]", "Url to eth1 rpc node")
+      .option("--rpc [api]", "Exposes the selected RPC api, must be comma separated")
+      .option("-c, --configFile [config_file]", "Config file path")
+
+      .option("-k, --key [key]", "Private key of the validator")
+      .option("--dbValidator [db_path]", "Path to the validator database")
+      .option(`-l, --loggingLevel [${Object.values(LogLevel).join("|")}]`, "Logging level")
+      .action(async (options) => {
+        // library is not awaiting this method so don't allow error propagation
+        // (unhandled promise rejections)
+        try {
+          let nodeCommand = new BeaconNodeCommand();
+          await nodeCommand.action(options, logger);
+          await this.action(options, nodeCommand.node, logger);
+        } catch (e) {
+          logger.error(e.message + '\n' + e.stack);
+        }
+      });
+  }
+
+  public async action(options: IStartCommandOptions, node: BeaconNode, logger: ILogger): Promise<void> {
+    if (options.loggingLevel) {
+      logger.setLogLevel(LogLevel[options.loggingLevel]);
+    }
+
+    let dbName: string;
+    if (options.dbValidator) {
+      dbName = options.dbValidator;
+    } else {
+      dbName = defaults.db.name;
+    }
+    let db = new ValidatorDB({
+      controller: new LevelDbController({
+        name: dbName
+      }, {
+        logger: logger
+      })
+    });
+
+    const rpcClient = new RpcClientOverInstance({
+      beacon: new BeaconApi({}, {chain: node.chain, db: node.db}),
+      validator: new ValidatorApi({},{chain:node.chain, db: node.db, opPool: node.opPool})
+    });
+
+    let keypair: Keypair;
+    if(options.key){
+      if (fs.existsSync(options.key)) {
+        keypair = await this.getKeyFromKeyStore(options.key);
+      } else {
+        keypair = new Keypair(PrivateKey.fromHexString(options.key));
+      }
+    } else {
+      throw new Error("Provide keystore file path or private key.");
+    }
+
+    let validatorCtx: ValidatorCtx = {
+      rpc: rpcClient,
+      keypair: keypair,
+      db: db,
+    };
+
+    let validator = new Validator(validatorCtx, logger);
+    await validator.start();
+  }
+
+  public async getKeyFromKeyStore(keyStorePath: string): Promise<Keypair> {
+    const password = await promptPassword("Enter password to decrypt the keystore: ");
+
+    const keystore = Keystore.fromJson(keyStorePath);
+    return keystore.getKeypair(password);
+  }
+
+}
```

Comment:
Where does the beacon chain get initiated ?

Reply:
I'm not sure how I feel about running both at the same time

Reply:
The beacon chain get initiated [here](https://github.com/ChainSafe/lodestar/blob/ddab426eeacc1eb5a07d931e968142ca6d2859de/src/cli/commands/start.ts#L57)

Reply:
I need to think through this flow - give me another day.